tools/dgeni/src/templates/api/base.template.html

```diff
@@ -2,5 +2,7 @@
 <h1>{$ doc.name $}</h1>
 {% endblock %}
 <p>{$ doc.description $}</p>
+<h2>API</h2>
```

Comment:
I don't think it's necessary to have this heading?

Reply:
While I'm not attached to this specific heading, I think some separator between the description and the source block is necessary. When the descriptions includes some code blocks at the end (which is fairly common), I think its not immediately apparent that the source block is not part of the description. See `/docs/api/auth/state/daffAuthProvideUnauthenticatedHooks`:
![image](https://github.com/user-attachments/assets/772ed2ba-93ec-445c-9d84-fdb36b180344)

Reply:
Yeah, I agree there should be a separator then. Is the `providers` code block a usage example? If so, I think we should also label it as usage example as well, or split up usage example and api into tabs like how angular does it?

Reply:
yes the provider block is a usage example. I agree, we should tag it as [`@example`](https://jsdoc.app/tags-example)

Reply:
> yes the provider block is a usage example. I agree, we should tag it as [`@example`](https://jsdoc.app/tags-example) and render it separately (tabs sound good)

Sounds good. @damienwebdev likes the idea of tabs as well.